packages/venia-concept/src/components/CategoryList/__tests__/categoryList.spec.js

```diff
@@ -85,7 +85,7 @@ test('displays category tiles', async () => {
                                 id: 17,
                                 name: 'baz',
                                 url_key: 'baz-url.html',
-                                url_path: '/bar-url.html',
+                                url_path: '/baz-url.html',
```

Comment:
I'll never understand why the first three default "random" words include two that look so alike.